docs/src/gettingstarted/introduction/own-code/project-configuration.md

```diff
@@ -29,20 +29,32 @@ In the previous step, you created a new project on Platform.sh using the CLI. No
     └── < application code >
     ```
 
-    In order to successfully deploy to Platform.sh you must add three YAML files:
+    In order to successfully deploy to Platform.sh you need to add two YAML files:
 
-      * A `.platform/routes.yaml` file, which configures the [routes](/configuration/routes/_index.md) used in your environments. That is, it describes how an incoming HTTP request is going to be processed by Platform.sh.
-      * A `.platform/services.yaml` file, which configures the [services](/configuration/services/_index.md) that will be used by the application. Connecting to Platform.sh's maintained services only requires properly writing this file. While this file must be present, if your application does not require services it can remain empty.
-      * At least one file [configuring the application](../../../configuration/app/_index.md). It provides control over the way the application will be built and deployed on Platform.sh.
+      * A `.platform/routes.yaml` file, which configures the [routes](/configuration/routes/_index.md) used in your environments.
+        That is, it describes how an incoming HTTP request is going to be processed by Platform.sh.
+      * At least one file [configuring the application](../../../configuration/app/_index.md).
```

Comment:
It doesn't actually need to be a `.platform.app.yaml` file, since `.platform/applications.yaml` is possible. That's why I offloaded the details to another page that should answer questions about details.

Reply:
It's best to avoid [plurals in parentheses](https://developers.google.com/style/plurals-parentheses?hl=en). The exact way to word it can be figured out when we redo the page.